INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_cha/TGT_repair_mutation_for_less_line_118.sol:MyAdvancedToken:
INFO:symExec:Running, please wait...
Traceback (most recent call last):
  File "_ctypes/callbacks.c", line 315, in 'calling callback function'
  File "/root/osiris/z3/z3.py", line 137, in z3_error_handler
    def z3_error_handler(c, e):
  File "/root/osiris/symExec.py", line 2868, in handler

  .oooooo.             o8o            o8o          
 d8P'  `Y8b            `"'            `"'          
888      888  .oooo.o oooo  oooo d8b oooo   .oooo.o
888      888 d88(  "8 `888  `888""8P `888  d88(  "8
888      888 `"Y88b.   888   888      888  `"Y88b. 
`88b    d88' o.  )88b  888   888      888  o.  )88b
 `Y8bood8P'  8""888P' o888o d888b    o888o 8""888P'


!!! SYMBOLIC EXECUTION TIMEOUT !!!
    raise Exception("timeout")
Exception: timeout
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 56.6%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 50.106621027 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_cha/TGT_repair_mutation_for_less_line_118.sol:owned:
INFO:symExec:Running, please wait...
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 99.4%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 0.307389974594 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_cha/TGT_repair_mutation_for_less_line_118.sol:token:
INFO:symExec:Running, please wait...
Traceback (most recent call last):
  File "/root/osiris/taintFlow.py", line 652, in perform_taint_analysis
    propagate_taint(taint, tainted_stack, tainted_memory, tainted_storage, instruction, current_stack, previous_block, current_block, next_blocks, arithmetic_errors, sha3_list, false_positives, strings)
  File "/root/osiris/taintFlow.py", line 416, in propagate_taint
    if remove_line_break_space(sha3) == data and not arithmetic_error["pc"] in false_positives:
  File "/root/osiris/symExec.py", line 2868, in handler
    raise Exception("timeout")
Exception: timeout
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 82.5%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 50.0103712082 seconds ---
INFO:symExec:	====== Analysis Completed ======


!!! SYMBOLIC EXECUTION TIMEOUT !!!
Unexpected error: <type 'exceptions.Exception'>